src/vendor/Path.ts

```diff
@@ -276,7 +276,7 @@ export default class Path {
 
   mkdir(): Path {
     if (!this.isDirectory()) {
-      Deno.mkdirSync(this.string)
+      Deno.mkdirSync(this.string, { recursive: true })
```

Comment:
but we have `mkpath()` for that.

while I agree I don't particularly see many reasons to not always `mkdir -p` I feel we should meet expectations and based on history that is mkdir doesn't recurse.

Reply:
I hate that feature of `mkdir`, but it feels safer for it to be explicit.

Reply:
like really we should have `mkdir` and have it have a `p` option. Using the string `'p'` because I have oddly become attached to the idea of this library mirroring the cli commands completely.

Reply:
[you're not the only one](https://www.npmjs.com/package/mkdirp)